src/SIM/SIMbase.C

```diff
@@ -530,6 +538,9 @@ bool SIMbase::parse (const TiXmlElement* elem)
     else if (!strcasecmp(elem->Value(),"discretization"))
       result &= opt.parseDiscretizationTag(child);
 
+  if (myGen)
+    myGen->createTopology(*this);
+
```

Comment:
Hvis createTopology metoden her forutsettes å kalle createFEMmodel() for sim'en, slik det gjøres ved parsing av "topology" tag-en, så kan dette kallet flyttes inne i blokka over, f.eks. like etter linje 517 og gen-objektet kan være lokal for den blokka (ikke noe stort poeng, men lokaliserer genereringen ytterligere)